src/main/java/wolox/training/exceptions/handler/ErrorsEnum.java

```diff
@@ -3,7 +3,8 @@
 public enum ErrorsEnum {
 
     JSON_BOOK_NOT_FOUND_EXCEPTION("404", "Book Not Found"),
-    JSON_BOOK_ID_MISMATCH_EXCEPTION("404", "Book Id Not Found");
+    JSON_BOOK_ID_MISMATCH_EXCEPTION("400", "Book Id Not Found"),
+    JSON_BOOK_ALREADY_OWNED_EXCEPTION("404", "Book Already Owned");
```

Comment:
The code 404 represent "not found data" and not represent "bad request".
Check the error handler.